charts/redpanda/templates/statefulset.yaml

```diff
@@ -65,23 +59,19 @@ spec:
       initContainers:
         - name: set-datadir-ownership
           image: busybox:latest
-          {{- $uid := dig "podSecurityContext" "runAsUser" .Values.statefulset.securityContext.runAsUser .Values.statefulset }}
```

Comment:
thank you for that, its been annoying me and was going to change this on a nit PR.